lib/galaxy/webapps/galaxy/controllers/user.py

```diff
@@ -788,7 +788,8 @@ def __register( self, trans, cntrller, subscribe_checked, **kwd ):
                     subject = 'Join Mailing List'
                     try:
                         util.send_mail( frm, to, subject, body, trans.app.config )
-                    except:
+                    except Exception, e:
+                        log.error( 'Subscribing to the mailing list has failed.', e )
```

Comment:
For a generic exception like this, `log.exception` is probably better since it will log the traceback.